### DISCUSSION TITLE
Getting Error at pom.xml

### DISCUSSION THREAD

**Discussion by vishwabujji** [akash-coded/spring-framework#77] [Q&A]
![Screenshot (77)](https://user-images.githubusercontent.com/101938974/168481539-60e61724-3b14-4bf8-8213-6fcb0d7ab484.png)

**Comment by akash-coded** [comment]
Spring Tools 4.14.1 is missing tm4e.

To install it, go to "Help → Install new software → Add URL: [https://download.eclipse.org/tm4e/snapshots/"](https://download.eclipse.org/tm4e/snapshots/%22). Click finish. Restart Eclipse.
